### PR TITLE
Add input for setting docker registry for microk8s

### PR DESCRIPTION
**Pull request by yhaliaw**
Add workflow input for microk8s to use private docker registry.

### CODE REVIEW

**Review by jdkandersson — APPROVED** [review]

**Review by jdkandersson — APPROVED** [review]

**Comment by addyess** [comment]
i'd REALLY like @neoaggelos to take a look at this but he may not have a moment to do so.

**Review by neoaggelos — CHANGES_REQUESTED** [review]
Instead of exec, please use File IO. Also, microk8s stop/start is not required, prefer `snap restart microk8s.daemon-containerd`, or nothing at all (the config applies immediately

File: src/bootstrap/index.ts [hunk 1]
```diff
@@ -151,6 +174,7 @@ async function run() {
     const microk8s_group = get_microk8s_group();
     let bootstrap_constraints = core.getInput("bootstrap-constraints");
     const microk8s_addons = core.getInput("microk8s-addons")
+    const microk8s_docker_registry = core.getInput("microk8s-docker-registry") || process.env["MICROK8S_DOCKER_REGISTRY"]
```
Comment (addyess) [inline src/bootstrap/index.ts:177]: @yhaliaw -- here's how to use  the action input or environment variable

Comment (yhaliaw) [reply src/bootstrap/index.ts]: Can `DOCKERHUB_MIRROR` be used as the env var instead?
I have updated the code.

File: src/bootstrap/index.ts [hunk 2]
```diff
@@ -151,6 +174,7 @@ async function run() {
     const microk8s_group = get_microk8s_group();
     let bootstrap_constraints = core.getInput("bootstrap-constraints");
     const microk8s_addons = core.getInput("microk8s-addons")
+    const microk8s_docker_registry = core.getInput("microk8s-docker-registry") || process.env["DOCKERHUB_MIRROR"]
```
Comment (addyess) [inline src/bootstrap/index.ts:177]: so? why is one `DOCKERHUB_MIRROR` and the other `microk8s-docker-registry`

feels to me these ought to be consistent.

Is the intent here to use a replacement oci-registry?

I kinda like either `container-registry-url` or `oci-registry-url` and just drop the "microk8s" bit.  
then have the env var `CONTAINER_REGISTRY_URL` or `OCI_REGISTRY_URL`

Comment (yhaliaw) [reply src/bootstrap/index.ts]: Updated the `microk8s-docker-registry` to `container-registry-url` in the code

**Review by jdkandersson — APPROVED** [review]

**Pull request merged**